lib/cosmos/tools/tlm_viewer/tlm_viewer.rb

```diff
@@ -67,9 +67,9 @@ def self.load_config(filename)
           else
             additional_data << target.original_name
           end
-          Dir.new(screen_dir).each do |screen_dir_filename|
-            if screen_dir_filename[0..0] != '.'
-              tlmviewer_files << File.join(screen_dir, screen_dir_filename)
+          Dir.new(screen_dir).each do |filename|
```

Comment:
using filename here mirrors the filename function parameter,  revert to screen_dir_filename